CHANGES.md

```diff
@@ -1,3 +1,44 @@
+## edge-19.8.2
+
+**Significant Update**
+
+This edge release introduces the new Linkerd control plane Helm chart, named
+linkerd2. Helm users can now install and remove the Linkerd control plane by
+using the `helm install` and `helm delete` commands. Proxy injection also now
+uses Helm charts.
+
+No changes were made to the existing `linkerd install` behavior.
+
+For detailed installation steps using Helm, see the notes for [PR #3146]
+(https://github.com/linkerd/linkerd2/pull/3146).
+
+* CLI
+  * Moved `linkerd top` and `linkerd profile --tap` to the new tap APIService
+  * Introduced parent resources for all tappable subresources in the tap
+    APIService, enabling commands such as `kubectl auth can-i`
+* Controller
+  * Introduced a new ClusterRole, `linkerd-linkerd-tap-admin`, which gives
+    cluster-wide tap privileges. Also introduced a new ClusterRoleBinding,
+    `linkerd-linkerd-web-admin`, which binds the `linkerd-web` service account
+    to the new tap ClusterRole. This ClusterRoleBinding is enabled by default,
+    but may be disabled via a new `linkerd install` flag,
+    `--restrict-dashboard-privileges`
+  * Added the ability to set a custom cluster domain in service profiles (thanks
+    @arminbuerkle!)
+  * Removed successfully completed `linkerd-heartbeat` jobs from pod listing in
+    the linkerd control plane to streamline `get po` output (thanks
+    @Pothulapati!)
+* Proxy
```

Comment:
the most recent proxy bump i see in master is from Aug 1 (f146041c25f79b9203ca9c3750fabe0c14b5c7df), expected?

Reply:
The changes mentioned here would need us to be on at least linkerd/linkerd2-proxy@f9dd4f8e28b9a6d8cdbe8094368495d94441b741

Reply:
Just talked with @olix0r and I will not bump the proxy before this edge release, due to an integration test failure. I am removing all of the `Proxy` bullet points from these release notes as they all occur after the current proxy version.